### PR TITLE
Update international-labour-organization.csl

### PR DESCRIPTION
**Pull request by vl2**
Minor fixes.

### CODE REVIEW

File: international-labour-organization.csl [hunk 1]
```diff
@@ -158,12 +160,12 @@
             </group>
           </group>
         </else-if>
-        <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" " suffix="."/>
+        <else-if type="chapter" match="any">
+          <text macro="title" prefix=" &quot;" suffix="&quot;,"/>
```
Comment (rmzelle) [inline international-labour-organization.csl]: It's better to use the "quotes" attribute here (`quotes="true"`). See http://docs.citationstyles.org/en/stable/specification.html#quotes

File: international-labour-organization.csl [hunk 2]
```diff
@@ -190,17 +192,24 @@
...
               <text macro="publisher"/>
-              <text variable="volume"/>
+              <text variable="volume" text-case="capitalize-first"/>
               <text macro="pages"/>
             </group>
           </group>
         </else-if>
+        <else-if type="article-newspaper" match="any">
+          <group>
+            <text macro="title" prefix=" &quot;" suffix=".&quot;"/>
```
Comment (rmzelle) [inline international-labour-organization.csl]: It's better to use the "quotes" attribute here (`quotes="true"`). See http://docs.citationstyles.org/en/stable/specification.html#quotes

**Comment by adam3smith** [comment]
Please fix the two quotes issues mentioned by @rmzelle above, otherwise this is good to go. 
You can update your pull request by using the pencil icon here:
https://github.com/vl2/styles/blob/patch-3/international-labour-organization.csl
Let us know when you're done of if you need any help.

**Comment by rmzelle** [comment]
@vl2, if you could update this pull request, that would be great.

File: international-labour-organization.csl (hunk 1 above)
Comment (vl2) [inline international-labour-organization.csl]: Thanks for the validation. Just back from the holidays and heard there are
also some other changes needed.
I'll update the pull request with "quotes" + other modifications asap.

**Comment by rmzelle** [comment]
Closing for inactivity.

@vl2, we can still accept this pull request if you make the changes we requested above and post a comment here that you have done so.

**Pull request closed without merging**